sql/house.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `house` (
+	`LandSetId` INT UNSIGNED NULL,
+	`HouseId` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+	`BuildTime` BIGINT NULL,
+	`Aetheryte` SMALLINT NULL, 
+	`Welcome` SMALLINT NULL,  
+	`Comment` binary(193) DEFAULT "", 
+	`Like` BIGINT NULL, 
```

Comment:
keyword in sql, might need to quote in backticks when querying (or maybe rename to Likes/Endorsements?)